spswarehouse/general/selenium.py

```diff
@@ -1,53 +1,62 @@
+import time
+
 from selenium.webdriver.support.ui import WebDriverWait, Select
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as EC
 
 
+### Internal Functions
+
+def _wait_for_element_to_be_clickable_and_return_it(driver, by_object, target_identifier, wait_time_in_seconds=30):
```

Comment:
That seems more complicated when calling the function to me. It would just add extra parentheses in a way that doesn't seem that useful. People would probably call the function by passing `(driver, (by_object, target_identifier), wait_time)`.

Either way, this particular function is just used internally to this file, and I think the way it is currently is fairly straight-forward.